src/erlfmt.erl

```diff
@@ -234,7 +262,7 @@ format_file_range(FileName, StartLocation, EndLocation, Options) ->
     string(),
     erlfmt_scan:location(),
     erlfmt_scan:location(),
-    [{print_width, pos_integer()}]
+    config()
```

Comment:
The options `verbose` and `pragma` are not supported by this function, that is why this is a smaller property list.
Could we change this back?

Reply:
So, the range was extracted for dispatching purpose, but still sits in this properly list. Would I rather:
* Remove it from the properly list? Con: that's additional work just to 
* Introduce a sub_config() type with just `print_width` and `range`?
* Other. Con: I have no idea what this alternative is.

Reply:
We want the function spec to only support what the function actually can support and not say that it can actually influence verbosity or where the output goes.

Reply:
Done :)